test/vaadin-list-mixin.html

```diff
@@ -479,6 +478,64 @@
           expect(list._scrollerElement.scrollTop).to.be.greaterThan(0);
         });
       });
+
+      describe('Multiple', () => {
+        it('should clear selected when multiple=true', () => {
+          list.selected = 3;
+          list.multiple = true;
+          expect(list.selected).to.be.equal(undefined);
+        });
+
+        it('should move selected to selectedValues when multiple=true', () => {
+          list.selected = 3;
+          list.multiple = true;
+          expect(list.selectedValues).to.eql([3]);
+        });
+
+        it('should clear selectedValues when multiple=false', () => {
+          list.multiple = true;
+          list.selectedValues = [3];
+          list.multiple = false;
+          expect(list.selectedValues).to.eql([]);
+        });
+
+        it('should reset selected items when multiple=false', () => {
+          list.multiple = true;
+          list.selectedValues = [1, 3];
+          list.multiple = false;
+          expect(list.items.filter(item => item.selected).length).to.eql(0);
+        });
+
+        it('should set selectedValues when clicking item', () => {
+          list.multiple = true;
+          list.items[3].click();
+          expect(list.selected).to.be.equal(undefined);
+          expect(list.selectedValues).to.eql([3]);
+        });
+
+        it('should set selected when clicking item and multiple=false', () => {
+          list.items[3].click();
+          expect(list.selected).to.be.equal(3);
+          expect(list.selectedValues).to.eql([]);
+        });
+
+        it('should add item to the selectedValues when clicking', () => {
+          list.multiple = true;
+          list.items[1].click();
+          expect(list.selectedValues).to.eql([1]);
+          list.items[3].click();
+          expect(list.selectedValues).to.eql([1, 3]);
+        });
+
+        it('should remove item from the selectedValues when clicking', () => {
```

Comment:
Should probably have also tests for `item.selected` set and removed on click (or add those checks to these test cases if you wish)

Reply:
Updated existing tests.